fs/_ftp_parse.py

```diff
@@ -138,12 +145,22 @@ def decode_linux(line, match):
     return raw_info
 
 
+def _decode_windowsnt_time(date, time):
```

Comment:
Might be worth renaming these parameters to `mdate` and `mtime`, to avoid potential clashes with the `time` module-name?

Reply:
Done